core-processor/src/configs.rs

```diff
@@ -18,15 +18,21 @@
 
 //! Configurations.
 
+use crate::common::ExecutableActor;
 use alloc::collections::BTreeSet;
 use codec::{Decode, Encode};
-use gear_core::{costs::HostFnWeights, memory::WasmPageNumber};
+use gear_core::{
+    costs::HostFnWeights, ids::ProgramId, memory::WasmPageNumber, message::IncomingDispatch,
+};
 
 const MAX_WASM_PAGES: u32 = 512;
 const INIT_COST: u64 = 5000;
 const ALLOC_COST: u64 = 10000;
 const MEM_GROW_COST: u64 = 10000;
 const LOAD_PAGE_COST: u64 = 3000;
+const EXISTENTIAL_DEPOSIT: u128 = 500;
+const OUTGOING_LIMIT: u32 = 1024;
+const MAILBOX_THRESHOLD: u64 = 3000;
```

Comment:
We should avoid such consts here. These params should be a runtime constants or mutable params there